client/src/components/v3/status/WMStatus.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { Col, Card, Accordion, Button, Table } from 'react-bootstrap';
+import OnlineStatusPill, {
+  isOnlineT,
+} from 'components/common/OnlineStatusPill';
+import { SensorDataT } from 'types/data';
+
+export interface WMStatusProps {
+  moduleName: string;
+  isOnline: isOnlineT;
+  data: SensorDataT[];
+  batteryVoltage?: number;
+  mqttAddress?: string;
+}
```

Comment:
Do a similar thing for this type as suggested for the camera.

Reply:
This case is different from camera as this is using optional types rather than required union types.
5545051